flocker/node/test/test_loop.py

```diff
@@ -36,16 +38,16 @@
     NodeState, Deployment, Manifestation, Dataset, DeploymentState,
     Application, DockerImage,
 )
-from ...control._protocol import NodeStateCommand, AgentAMP
+from ...control._protocol import NodeStateCommand, AgentAMP, SetNodeEraCommand
 from ...control.test.test_protocol import iconvergence_agent_tests_factory
 from .. import NoOp
 
 
 def build_protocol():
     """
-    :return: ``Protocol`` hooked up to transport.
+    :return: ``AMP`` hooked up to transport.
     """
-    p = Protocol()
+    p = AMP()
     p.makeConnection(StringTransport())
     return p
```

Comment:
:grey_question: This is now duplicated, here and in test_protocol. Consider moving to a testtools module and importing.